### PR TITLE
Port to GH Actions and Packages

### PR DESCRIPTION
**Pull request by n1v0lg**
Migrating from Travis to Github Actions and from Maven Central to Github Packages.

### CODE REVIEW

File: .github/workflows/build-n-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            ${{ runner.os }}-gradle-
+      - name: Set up Java for publishing to GitHub Packages
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Gradle Setup
+        run: |
+          echo "
+          mavenUser=${{secrets.GRADLE_USER}}
+          mavenPassword=${{secrets.GRADLE_PASSWORD}}" >> gradle.properties
```
Comment (pastjean) [inline .github/workflows/build-n-deploy.yml:24]: Gradle Setup 👆 step is not needed :)

Comment (n1v0lg) [reply .github/workflows/build-n-deploy.yml]: @pastjean You sure? removing that breaks the build.

Comment (n1v0lg) [reply .github/workflows/build-n-deploy.yml:24]: Weird because both those values are blank.

File: build.gradle [hunk 2]
```diff
@@ -53,20 +46,44 @@ subprojects {
...
+    repositories {
+        mavenLocal()
+
+        maven {
+            credentials {
+                username "$mavenUser"
+                password "$mavenPassword"
+            }
+            url "https://maven.pleo.io/artifactory/libs-release"
+        }
```
Comment (pastjean) [inline build.gradle:58]:
```suggestion
```

Comment (pastjean) [reply build.gradle:58]: I am sure this refers to pleo private repo, in a public package this doesn't make sens

File: .github/workflows/build-n-deploy.yml (hunk 1 above)
Comment (pastjean) [reply .github/workflows/build-n-deploy.yml:24]: yes :) look at my next comment

Comment (n1v0lg) [reply .github/workflows/build-n-deploy.yml]: nice. thanks!!

File: README.md [hunk 3]
```diff
@@ -8,14 +8,10 @@ It is made of 4 modules that when combined give you a full, flexible and powerfu
...
-If you're okay with using Guice, Archaius and Jackson, add a dependency on `prop-all`.
-
-```xml
-<dependency>
-    <groupId>io.pleo</groupId>
-    <artifactId>prop-all</artifactId>
-    <version>1.2.0</version>
-</dependency>
+If you're okay with using Guice, Archaius and Jackson, add a dependency on `prop-all` (Gradle):
+
```
Comment (pastjean) [inline README.md:12]: i'd may add a note on how to include "repository" from gh package registry. But not mandatory.

**Pull request merged**